lib/rgl/bidirectional_adjacency.rb

```diff
@@ -52,8 +52,8 @@ def has_in_edge?(u, v)
     alias :has_out_edge? :has_edge?
 
     # @see BidirectionalGraph#each_in_neighbor
-    def each_in_neighbor(v)
-      @reverse.each_adjacent(v)
+    def each_in_neighbor(v, &b)
```

Comment:
Puh, I should have been more careful in my previous review of #76. Thank you for finding this!